test/unit/math/fwd/core/fvar_test.cpp

```diff
@@ -25,40 +27,9 @@ TEST(mathFwdCoreFvar, ctor) {
   fvar<double> c(1.93, -27.832);
   EXPECT_FLOAT_EQ(1.93, c.val_);
   EXPECT_FLOAT_EQ(-27.832, c.d_);
-
-  fvar<double> d = -c;
-  EXPECT_FLOAT_EQ(-1.93, d.val_);
-  EXPECT_FLOAT_EQ(27.832, d.d_);
-
-  fvar<double> e(5.0);
-  d += e;
-  EXPECT_FLOAT_EQ(3.07, d.val_);
-
-  EXPECT_FLOAT_EQ(3.07, (d++).val_);
-  EXPECT_FLOAT_EQ(4.07, d.val_);
-
-  EXPECT_FLOAT_EQ(5.07, (++d).val_);
-  EXPECT_FLOAT_EQ(5.07, d.val_);
-
-  double nan = std::numeric_limits<double>::quiet_NaN();
-  fvar<double> f(nan);
```

Comment:
Isn't this the only test that needs removed? Or do all the other tests just seem extraneous?

Reply:
I'm trying to reduce tests to just testing inputs with finite differences.  Was there a test you think is missing?  

I could just leave all the old test code, but that seems confusing.

Reply:
It seems to be testing operator-, operator+=, operator++ or whatnot? If these are tested elsewhere that's cool